### PR TITLE
WIP: View and edit repository documentation 

### PR DESCRIPTION
**Pull request by mosabua**
https://issues.sonatype.org/browse/NEXUS-8404

Do not review this yet... @kellyrob99 recommended to put it on hold until a couple of the bugs found are ironed out and the UI refresh is done. I will pick up next week.

### CODE REVIEW

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -217,13 +217,18 @@ from the local storage therefore eliminating the network and time
 overhead of retrieving the component from the remote repository again.
 
 By default, Nexus ships with the following configured proxy
-repositories in Maven2 format:
```
Comment (jwayman) [inline chapter-configuration.asciidoc]: configured, proxy

File: chapter-configuration.asciidoc [hunk 2]
```diff
@@ -217,13 +217,18 @@ from the local storage therefore eliminating the network and time
...
+repositories:
 
-Apache Snapshots:: This repository contains snapshot releases from the
-Apache Software Foundation.
+maven-central:: This proxy repository accesses the
+http://search.maven.org/[Central Repository], also known as Maven
+Central. It is the default component repository built into Apache
+Maven and is well supported by other build tools like Gradle, SBT or
```
Comment (jwayman) [inline chapter-configuration.asciidoc]: well-supported

Comment (jdillon) [inline chapter-configuration.asciidoc]: When did this get rebranded as "The Central Repository"?  Its always been "Maven Central" AFAICR or "central" for short for folks who only care about Maven.

Comment (mosabua) [inline chapter-configuration.asciidoc]: Not sure.. probably a year or two ago. I don't remember exactly but it feels like a LONG time ago and has been in the Nexus docs, the website as well as the sites for Central themselves.

E.g. http://search.maven.org/ or http://central.sonatype.org/

**Comment by mosabua** [comment]
Too many conflicts and painful to merge. I am respinning this PR from a different branch.

**Pull request closed without merging**